todoman/cli.py

```diff
@@ -52,13 +52,13 @@ def _todo_property_options(command):
         help=('The due date of the task, in the format specified in the '
               'configuration file.'))(command)
     click.option(
-        '--start', '-s', default='', callback=_validate_date_param,
+        '--dtstart', '-s', default='', callback=_validate_date_param,
```

Comment:
I'd rather call this `start`. Even if vCard field name is `dtstart`, I think that for most users, start is more understandable (we *understand* vCard stuff, but most don't).